src/components/router/router.js

```diff
@@ -40,7 +39,7 @@ class Router extends React.Component {
               path={item.to}
               strict={item.strict}
               render={({ location, ...routeProps }) => {
-                const navDetail = routerHelpers.getNavigationDetail({
+                const navDetail = routerHelpers.getNavRouteDetail({
```

Comment:
review: is this necessary anymore? this made sense when nav and route were separated but it appears we can just dump the route config props in at the same time instead of calling them after-the-fact. there is a minor mapping for the product and view params to consider

Reply:
maintaining with a minor rename for now, review once context and hooks start taking over. review using `react-router-dom`s `useRouteMatch`